[2023-02-14T14:18:00.796+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_v1_05.insert_into_postgres_table manual__2023-02-14T14:17:59.155057+00:00 [queued]>
[2023-02-14T14:18:00.802+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_v1_05.insert_into_postgres_table manual__2023-02-14T14:17:59.155057+00:00 [queued]>
[2023-02-14T14:18:00.803+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-14T14:18:00.803+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-02-14T14:18:00.804+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-14T14:18:00.811+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_into_postgres_table> on 2023-02-14 14:17:59.155057+00:00
[2023-02-14T14:18:00.815+0000] {standard_task_runner.py:55} INFO - Started process 10470 to run task
[2023-02-14T14:18:00.818+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag_v1_05', 'insert_into_postgres_table', 'manual__2023-02-14T14:17:59.155057+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpxvm1ycff']
[2023-02-14T14:18:00.820+0000] {standard_task_runner.py:83} INFO - Job 101: Subtask insert_into_postgres_table
[2023-02-14T14:18:00.870+0000] {task_command.py:388} INFO - Running <TaskInstance: postgres_operator_dag_v1_05.insert_into_postgres_table manual__2023-02-14T14:17:59.155057+00:00 [running]> on host 01550fe0586b
[2023-02-14T14:18:00.934+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tlysenko
AIRFLOW_CTX_DAG_ID=postgres_operator_dag_v1_05
AIRFLOW_CTX_TASK_ID=insert_into_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2023-02-14T14:17:59.155057+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-14T14:17:59.155057+00:00
[2023-02-14T14:18:00.935+0000] {sql.py:254} INFO - Executing: 
            INSERT INTO dag_runs(dt, dag_id) values('2023-02-14', 'postgres_operator_dag_v1_05')
        
[2023-02-14T14:18:00.945+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-02-14T14:18:01.044+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-02-14T14:18:01.049+0000] {sql.py:375} INFO - Running statement: 
            INSERT INTO dag_runs(dt, dag_id) values('2023-02-14', 'postgres_operator_dag_v1_05')
        , parameters: None
[2023-02-14T14:18:01.052+0000] {sql.py:384} INFO - Rows affected: 1
[2023-02-14T14:18:01.060+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag_v1_05, task_id=insert_into_postgres_table, execution_date=20230214T141759, start_date=20230214T141800, end_date=20230214T141801
[2023-02-14T14:18:01.114+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-14T14:18:01.130+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
